clouds/aws/aws_roles/aws_roles.py

```diff
@@ -0,0 +1,67 @@
+import botocore.errorfactory
+from aws.aws_clients import IAMClient
+from botocore.exceptions import BotoCoreError
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+class AWSRoles:
+    def __init__(self, client: IAMClient):
+        """
+        Manages AWS roles and role-policies.
+
+        Args:
+            client (IAMClient): AWS roles managements expects an IAM client entity
+        """
+        self.client = client
+
+    def create_role_policy(self, role_name, policy_name, policy_json_path):
+        """
+        Create the role policy with specific configuration.
+
+        Args:
+            role_name (str): role policy name
+            policy_name (str): policy name
+            policy_json_path (str): path to json file that holds the policy documents
+
+        Raises:
+            If POST request failed
+        """
+        LOGGER.info(f"Creating new role {role_name} for policy {policy_name}.")
+        if not self.check_if_role_policy_exists(role_name=role_name):
+            try:
+                with open(policy_json_path, "r") as fd:
+                    policy_doc = fd.read()
```

Comment:
if this is a json file please use json to read it.

Reply:
Tried at first, the policy update expects a string which holds **Json format content**, not a parsed (loaded) Json.
example input file:
'{ "Policy":"policy_content"}'